client/js/lib/core/libs/index.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
Do we really need the index file here? I don't know for sure.

Reply:
I find this better and cleaner for Path, like in `client/js/lib/core/loadLibs.js`

Reply:
agreed!